test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities_pagination.test

```diff
@@ -35,7 +35,7 @@ brokerStart CB
 # 03. POST /v2/entities, created third  entity
 # 04. POST /v2/entities, created fourth entity
 # 05. POST /v2/entities, created fifth  entity
-# 06. GET  /v2/entities?limit=2&count=true
+# 06. GET  /v2/entities?limit=2&options=count,normalized
```

Comment:
Probably 'normalized' will not be used at the end after the end, but it is ok to use it in this test as "random" option to check that the ability to mix count with other options works.

NTC